cl/recap/tasks.py

```diff
@@ -1419,3 +1428,71 @@ def mark_fq_status(fq, msg, status):
     if status == PROCESSING_STATUS.SUCCESSFUL:
         fq.date_completed = now()
     fq.save()
+
+
+@app.task(bind=True, max_retries=5, interval_start=5 * 60)
+def process_recap_email(self: Task, pk: int) -> Optional[int]:
+    start_time = now()
+    epq = EmailProcessingQueue.objects.get(pk=pk)
+    mark_pq_status(epq, "", PROCESSING_STATUS.IN_PROGRESS, "status_message")
+
+    message_id = epq.message_id
+    bucket = RecapEmailSESStorage()
+    body = bucket.open(message_id, "r").read()
```

Comment:
I haven't used this pattern before, but do you need to close your connection to the bucket? If so, can a context handler be used? Something like:

```
with bucket.open(...) as f:
    foo = f.read()
```

Reply:
Yes, I'll go with the "with" pattern, seems a little more Pythonic.